cli/cliui/prompt.go

```diff
@@ -22,6 +22,7 @@ type PromptOptions struct {
 	Text      string
 	Default   string
 	Secret    bool
+	Mask      rune
```

Comment:
Honestly we should just make it implicit that we'll use `*` as a mask when `Secret: true`. I can't think of any place where we wouldn't want asterisks to be echoed, nor can I think of any place where we would want `Secret: true, Mask: 0`